content/events/2020-campinas/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Campinas
+date: '2020-04-04'
+location:
+  country: Brasil
+  city: Campinas
+  countrycode: br
+  url: https://ima.sp.gov.br/
+  venue: Teatro IMA Cultural
+map: ima-en_us.jpg
+sched_slug: kubernetecommunitydayscampinas
+social:
+  twitter: https://twitter.com/kcdcampinas
+sponsors:
+- level: Initial sponsors
+  orgs:
+  - name: Mandic
+    logo: sponsors/mandic.svg
+    url: https://www.mandic.com.br/
+  - name: PagSeguro
+    logo: sponsors/pagseguro.png
+    url: https://www.pagseguro.com.br
+  - name: Aqua
+    logo: sponsors/aquasec.png
+    url:  https://www.aquasec.com/
+speakers:
+# - name: Phippy
+#   image: team/phippy-color.png
+#   twitter: https://twitter.com/cloudnativefdn
+#   bio: Phippy is a simple PHP app, trying to find a home in a cloud native world.
+#   homepage: https://phippy.io
+organizers:
+- name: Paulo Simões
+  image: team/paulosimoes.jpeg
+  twitter: https://twitter.com/pasimoes
+  homepage: https://www.linkedin.com/in/paulosimoes/
+- name: Eduardo Neves - eth0
+  image: team/eduardo.jpeg
+  twitter: https://twitter.com/_eth0_
+  homepage: https://www.linkedin.com/in/eduardo-neves/
+- name: Christiano Linuxmen
+  image: team/linuxmen.jpeg
+  twitter: https://twitter.com/linuxmen
+  homepage: https://www.linkedin.com/in/linuxmen/
+- name: Gabriela Dias
+  image: team/gaby.jpg
+  twitter: https://twitter.com/gabyldias
+  homepage: https://github.com/floord
+- name: Carol Vilas Boas
+  image: team/carol.jpeg
+  twitter: https://twitter.com/cavillasboas
+  homepage: https://www.linkedin.com/in/carolfvb/
+organizers_email: organizers-campinas@kubernetescommunitydays.org
+local_law_enforcement: 190
+medical_emergency: 193
+cfp_deadline: '2020-03-13'
```

Comment:
Please add the link to your CFP: https://kcdcfpsubmissions.smapply.io/prog/kcd_campinas/. I am reaching out with instructions on admin access. Thanks!

Reply:
Kristi as we started before CNCF setting up all tools, we already opened CFP on papercall.io .

We are very close to end date. We really should change?

https://www.papercall.io/kcd-campinas

Reply:
No problem- feel free to use Papercall given the short timeline. Thank you for your understanding.